menyer/enmeny/enmeny.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="sv">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style1.css">
+    <title>En av alla menyer</title>
+</head>
+
+<body>
+    <center>
+        <header class="header">
+            <div class="enmeny">
```

Comment:
Consider using English for the code part of the website.

And whatever you choose, stick with it.